include/seqan/basic/triple_packed.h

```diff
@@ -61,7 +61,7 @@ namespace seqan {
  * @tparam T3 The third type of the Triple.
  */
 
-#ifdef PLATFORM_WINDOWS
+#ifdef STD_LIB_VS
     #pragma pack(push,1)
```

Comment:
this is a compiler feature I think. Can you research whether it is specific to COMPILER_MSVC and if yes adapt this? How do clang and icc handle this on windows?